x/qgb/types/valset.go

```diff
@@ -26,54 +20,6 @@ func NewValset(nonce, height uint64, members InternalBridgeValidators) (*Valset,
 	return &vs, nil
 }
 
-// SignBytes produces the bytes that celestia validators are required to sign
-// over when the validator set changes.
-func (v *Valset) SignBytes(bridgeID ethcmn.Hash) (ethcmn.Hash, error) {
-	vsHash, err := v.Hash()
-	if err != nil {
-		return ethcmn.Hash{}, err
-	}
-
-	// the word 'checkpoint' needs to be the same as the 'name' above in the checkpointAbiJson
-	// but other than that it's a constant that has no impact on the output. This is because
-	// it gets encoded as a function name which we must then discard.
-	bytes, err := InternalQGBabi.Pack(
-		"domainSeparateValidatorSetHash",
-		bridgeID,
-		VsDomainSeparator,
-		big.NewInt(int64(v.Nonce)),
-		big.NewInt(int64(v.TwoThirdsThreshold())),
-		vsHash,
-	)
-	// this should never happen outside of test since any case that could crash on encoding
-	// should be filtered above.
-	if err != nil {
-		panic(fmt.Sprintf("Error packing checkpoint! %s/n", err))
-	}
-
-	hash := crypto.Keccak256Hash(bytes[4:])
-	return hash, nil
-}
-
-// Hash mimics the 'computeValsetHash' function used by the qgb contracts by using
-// a Valset to compute the hash of the abi encoded validator set.
-func (v *Valset) Hash() (ethcmn.Hash, error) {
```

Comment:
[blocking question]
If we're keeping the `Valset` struct, why do we want to delete the hash function here?

Reply:
Yes, I always keep forgetting this could be used for slashing eventually. Also, could be used as a source of truth for the signatures.
Closing PR. 
Thanks a lot for reminding me